src/main/java/com/linecorp/armeria/common/ServiceInvocationContext.java

```diff
@@ -43,9 +45,35 @@
 
     /**
      * Returns the context of the invocation that is being handled in the current thread.
+     *
+     * @throws IllegalStateException if the context is unavailable in the current thread
```

Comment:
Good point - looking through the sdk docs, they're not consistent but generally seem to leave it out